### PR TITLE
Adapt `Animation` events to new structure

### PR DESCRIPTION
**Pull request by teoli2003**
This PR updates the `Animation` interface for new-style event pages. (see the [template](https://developer.mozilla.org/en-US/docs/MDN/Structures/Page_types/API_event_subpage_template))

The `on-*` event handler pages are redirected to the `eventname_event` pages.

`yarn content move Web/API/Animation/oncancel Web/API/Animation/cancel_event`
`yarn content move Web/API/Animation/onfinish Web/API/Animation/finish_event`
`yarn content move Web/API/Animation/onremove Web/API/Animation/remove_event`

BCD: mdn/browser-compat-data#14914

### CODE REVIEW

File: files/en-us/web/api/animation/cancel_event/index.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+## Syntax
+
+Use the event name in methods like {{domxref("EventTarget.addEventListener", "addEventListener()")}}, or set an event handler property.
+
+```js
+addEventListener('cancel', event => { })
+oncancel = event => { }
+```
+
+### Value
```
Comment (Elchi3) [inline files/en-us/web/api/animation/cancel_event/index.md:25]: This "Value" section is not in the template. I don't think it is useful.

File: files/en-us/web/api/animation/remove_event/index.md [hunk 2]
```diff
@@ -1,32 +1,39 @@
...
 ## Syntax
 
-```js
-var removeHandler = animation.onremove;
+Use the event name in methods like {{domxref("EventTarget.addEventListener", "addEventListener()")}}, or set an event handler property.
 
-animation.onremove = removeHandler;
+```js
+addEventListener('remove', event => { })
+remove = event => { }
```
Comment (Elchi3) [inline files/en-us/web/api/animation/remove_event/index.md:20]:
```suggestion
onremove = event => { }
```

File: files/en-us/web/api/animation/remove_event/index.md [hunk 3]
```diff
@@ -1,32 +1,39 @@
...
-  - web animations api
-browser-compat: api.Animation.onremove
+  - Event
+browser-compat: api.Animation.remove_event
 ---
-{{ SeeCompatTable() }}{{ APIRef("Web Animations") }}
+{{ APIRef("Web Animations") }}
 
-The {{domxref("Animation")}} interface's **`onremove`** property (from the [Web Animations API](/en-US/docs/Web/API/Web_Animations_API)) is the event handler for the {{event("remove")}} event. This event is sent when the animation is removed (i.e., put into an `active` replace state).
+The **`remove`** event of the {{domxref("Animation")}} interface is fore when the animation is removed (i.e., put into an `active` replace state).
```
Comment (Elchi3) [inline files/en-us/web/api/animation/remove_event/index.md:12]:
```suggestion
The **`remove`** event of the {{domxref("Animation")}} interface fires when the animation is removed (i.e., put into an `active` replace state).
```

File: files/en-us/web/api/animation/cancel_event/index.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+## Browser compatibility
+
+{{Compat}}
+
+## See also
+
+- [Web Animations API](/en-US/docs/Web/API/Web_Animations_API)
+- {{domxref("Animation")}}
+- The {{event("cancel")}} event
```
Comment (Elchi3) [inline files/en-us/web/api/animation/cancel_event/index.md:64]:
```suggestion
```

**Review by Elchi3 — CHANGES_REQUESTED** [review]
Thanks, some change requests but looks good overall.

Aside: I see a lot of whitespace changes... I thought we updated many files recently to fix these things. Is something wrong with your editor settings or has mdn/content's editorconfig changed or something?

File: files/en-us/web/api/animation/cancel_event/index.md (hunk 1 above)
Comment (teoli2003) [reply files/en-us/web/api/animation/cancel_event/index.md]: Sorry forgot this one.

**Comment by teoli2003** [comment]
> Thanks, some change requests but looks good overall.
> 
> Aside: I see a lot of whitespace changes... I thought we updated many files recently to fix these things. Is something wrong with your editor settings or has mdn/content's editorconfig changed or something?

I modified them: they were unicode non-breakable spaces and I just replaced them with regular spaces. Some editors put them all over the place when you create new pages. (I think mainly Microsoft Word).

**Review by Elchi3 — APPROVED** [review]

**Pull request merged**